appserver/web/weld-integration/src/main/java/org/glassfish/cdi/transaction/TransactionalInterceptorBase.java

```diff
@@ -96,72 +98,72 @@
     public static final String CDI_JTA_NOCOMPONENT = "AS-JTA-00003";
 
     @LogMessageInfo( message = "In MANDATORY TransactionalInterceptor",
-                     level = "INFO")
+                     level = "FINE")
     public static final String CDI_JTA_MANDATORY = "AS-JTA-00004";
 
     @LogMessageInfo( message = "In NEVER TransactionalInterceptor",
-                     level = "INFO")
+                     level = "FINE")
     public static final String CDI_JTA_NEVER = "AS-JTA-00005";
 
     @LogMessageInfo( message = "In NOT_SUPPORTED TransactionalInterceptor",
-                     level = "INFO")
+                     level = "FINE")
     public static final String CDI_JTA_NOTSUPPORTED = "AS-JTA-00006";
 
     @LogMessageInfo( message = "Managed bean with Transactional annotation and TxType of NOT_SUPPORTED " +
                                "called inside a transaction context. Suspending transaction...",
-                     level = "INFO")
+                     level = "FINE")
     public static final String CDI_JTA_MBNOTSUPPORTED = "AS-JTA-00007";
 
     @LogMessageInfo( message = "Managed bean with Transactional annotation and TxType of NOT_SUPPORTED " +
                                "called inside a transaction context.  Suspending transaction failed due to {0}",
-                     level = "INFO")
+                     level = "FINE")
     public static final String CDI_JTA_MBNOTSUPPORTEDTX = "AS-JTA-00008";
 
     @LogMessageInfo( message = "In REQUIRED TransactionalInterceptor",
-                     level = "INFO")
+                     level = "FINE")
     public static final String CDI_JTA_REQUIRED = "AS-JTA-00009";
 
     @LogMessageInfo( message = "Managed bean with Transactional annotation and TxType of REQUIRED " +
                         "called outside a transaction context.  Beginning a transaction...",
-                     level = "INFO")
+                     level = "FINE")
     public static final String CDI_JTA_MBREQUIRED = "AS-JTA-00010";
 
     @LogMessageInfo( message = "Managed bean with Transactional annotation and TxType of REQUIRED " +
                                "encountered exception during begin {0}",
-                     level = "INFO")
+                     level = "FINE")
```

Comment:
Do we want the messages like this (lines 131, 136, 141, 145, 150, 155, and 160) to be at FINE level? They look like error messages to me. Leaving at INFO or changing to WARNING might be more appropriate.